infra/base-images/base-runner/bad_build_check

```diff
@@ -301,6 +301,12 @@ function check_mixed_sanitizers {
   local result=0
   local CALL_INSN=
```

Comment:
Can bad_build_check be done on jazzer targets? If so, why disable it in test_all.py.
If not, why make these changes to this file?

Reply:
I'm not sure I fully understand your concern here: 
I am disabling the `check_mixed_sanitizers` check for a similar reason to why this is done for Python: the sanitizers are linked into the driver, not the individual fuzz target. 
`check_architecture` is disabled as there is no direct way to obtain the native dependencies used by a Java fuzz target. We could alternatively apply the check to all binaries in `$OUT` or have a convention for the names of native libraries that fuzz targets can depend on.
The other checks are still run.

Reply:
Sorry, please see my other comment.